### PR TITLE
health check unit tests

### PR DESCRIPTION
**Pull request by pixelsoccupied**
/cc @donpenney 
/cc @Missxiaoguo 
/cc @jc-rh

### CODE REVIEW

File: internal/healthcheck/healthcheck_test.go [hunk 1]
```diff
@@ -0,0 +1,687 @@
...
+			s.AddKnownTypes(configv1.GroupVersion, &configv1.ClusterVersion{})
+			s.AddKnownTypes(configv1.GroupVersion, &configv1.ClusterVersionList{})
+			s.AddKnownTypes(mcv1.GroupVersion, &mcv1.MachineConfigPoolList{})
+			s.AddKnownTypes(mcv1.GroupVersion, &mcv1.MachineConfigPool{})
+			s.AddKnownTypes(configv1.GroupVersion, &configv1.ClusterOperatorList{})
+			s.AddKnownTypes(configv1.GroupVersion, &configv1.ClusterOperator{})
+			s.AddKnownTypes(operatorsv1alpha1.SchemeGroupVersion, &operatorsv1alpha1.ClusterServiceVersionList{})
+			s.AddKnownTypes(operatorsv1alpha1.SchemeGroupVersion, &operatorsv1alpha1.ClusterServiceVersion{})
+			s.AddKnownTypes(configv1.GroupVersion, &configv1.Infrastructure{})
+			s.AddKnownTypes(v1.SchemeGroupVersion, &v1.Node{})
```
Comment (Missxiaoguo) [inline internal/healthcheck/healthcheck_test.go:677]: nit: I think these do not have to be called every time a testcase is executed. Perhaps move them out of the for loop or define them only once in an init function?

Comment (pixelsoccupied) [reply internal/healthcheck/healthcheck_test.go]: Yep! Updated to use init()

**Review by jc-rh — APPROVED** [review]

**Pull request merged**